### DISCUSSION TITLE
241122_금

### DISCUSSION THREAD

**Discussion by wonjeongjeong** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#45] [TIL]
- [x] 서버 남으면 Augmentation 실험
- [ ] soft voting 앙상블

**Comment by sejongmin** [comment]
- [x] hflip offline aug랑 albumentation aug 비교해보기
- [x] 이상치 처리하기
- [ ] PSNR 확인해서 중복 이미지 있는지 확인해보기

**Comment by june21a** [comment]
- [x] elastic transform 관련해서 실험을 진행해보기
- [x] dice loss 테스트 해보기
- [ ] smp에서 augmentation을 변형하기 쉽도록 바꿔 만들기
